ote/src/cljs/ote/app/controller/admin.cljs

```diff
@@ -280,6 +281,35 @@
                :loading? false
                :results response))
 
+  UpdateInterfaceRadioFilter
+  (process-event [{f :radio-filter} app]
```

Comment:
When each `cond` case evaluates the same symbol, `condp` simplifies the implementation:
```suggestion
  (process-event [{f :radio-filter} app]
   (condp = f
     :all
     (-> app
         (assoc-in [:admin :interface-list :filters :import-error] false)
         (assoc-in [:admin :interface-list :filters :db-error] false)
         (assoc-in [:admin :interface-list :filters :no-interface] false))
     :no-interface
     (-> app
         (assoc-in [:admin :interface-list :filters :import-error] false)
         (assoc-in [:admin :interface-list :filters :db-error] false)
         (assoc-in [:admin :interface-list :filters :no-interface] true))
     :db-error
     (-> app
         (assoc-in [:admin :interface-list :filters :import-error] false)
         (assoc-in [:admin :interface-list :filters :db-error] true)
         (assoc-in [:admin :interface-list :filters :no-interface] false))
     :import-error
     (-> app
         (assoc-in [:admin :interface-list :filters :import-error] true)
         (assoc-in [:admin :interface-list :filters :db-error] false)
         (assoc-in [:admin :interface-list :filters :no-interface] false))
     ;; :else
     (-> app
         (assoc-in [:admin :interface-list :filters :import-error] false)
         (assoc-in [:admin :interface-list :filters :db-error] false)
         (assoc-in [:admin :interface-list :filters :no-interface] false)))
   )
```

Reply:
Or use `case` instead of the `condp` in the example I pasted :)

Reply:
I went with case and update-in. Thanx!